ecms_base/config/install/user.role.administrator.yml

```diff
@@ -2,7 +2,7 @@ langcode: en
 status: true
 dependencies: {  }
 id: administrator
```

Comment:
I think we should rename the id to `drupal_admin` to be consistent. The filename should also reflect that change.

We will want to update the `ecms_base.install` file to update the role assignment for user/1 too.